fastapi_router_controller/lib/controller.py

```diff
@@ -100,7 +100,12 @@ def __parse_controller_router(cls):
 
             # replace the class instance with the itself FastApi Dependecy
             signature_parameters[0] = signature_parameters[0].replace(default=Depends(cls))
-            new_signature = signature.replace(parameters=signature_parameters)
+            # set self and after it the keyword args
```

Comment:
@delijati Can you explain me with an example what this fix?

Reply:
An example is in the tests -> just remove the line above and you will see the error. Btw. take a look into `pytest` for testing